### ISSUE TITLE
I just wanted to say that it makes me sad that it's dead

### ISSUE THREAD

**Issue by julienreszka** [mozilla-extensions/firefox-voice#1913]

**Comment by Extarys** [comment]
Yeah I discovered this like 5 minutes ago on the [community portal](https://community.mozilla.org/en/activities/). Damn.

I'll start donating to Mozilla. Can't believe I never donated.

**Comment by grahamperrin** [comment]
I was not aware until I began seeking information in the absence of release notes for `0.25.0`: 

![image](https://user-images.githubusercontent.com/192271/106350782-7a078e80-62cf-11eb-9250-6083ac1447b1.png)

https://github.com/mozilla-extensions/firefox-voice/releases/tag/v0.25.0 does not exist. 

Re: https://addons.mozilla.org/addon/voice-fill/versions/ for a different (but related) extension, I assume that the release note for _Firefox Voice Beta_ `0.25.0` will be similar to the release note for _Voice Fill_ `1.4.7`:

> Implemented instrumentation to self-uninstall add-on

----

> … on the community portal. 

Please: where, exactly? 

[Mozilla Voice Server Proxy Retiring | Firefox Help](https://support.mozilla.org/en-US/kb/mozilla-voice-server-proxy-retiring) (undated) is quite vague: 

> … these extensions are being retired as part of the server decommissioning. …

– but there's no explanation for the decommissioning. I can guess a reason, but I'd prefer the explanation to come from Mozilla.

**Comment by grahamperrin** [comment]
Cross reference: 

* https://discourse.mozilla.org/t/retiring-the-voice-fill-and-firefox-voice-beta-extensions/74581/23?u=grahamperrin

**Comment by andrenatal** [comment]
> – but there's no explanation for the decommissioning. I can guess a reason, but I'd prefer the explanation to come from Mozilla.

Hi @grahamperrin. I'm a Mozilla employee and have created Voice Fill, Firefox Voice, our speech-proxy server, and numerous other voice projects here and was the one responsible for decommissioning the addons.

The reason is simple and transparent: we are decommissioning the addons purely because the use of Google's STT backend won't scale for us, and we never managed to train production ready models through projects like Deep Speech and Common Voice with the quality required for VF and FxVoice.

**Comment by Extarys** [comment]
@andrenatal I really appreciate you take the time to provide this explanation and it's totally understandable.

:heart:

**Comment by grahamperrin** [comment]
Likewise, @andrenatal much appreciated. 

(After shifting the discussion to Discourse, I found some key posts on the subject; linked/quoted there. It's most useful to have your comment here as well.)
